src/services/__tests__/AsyncStorage.spec.ts

```diff
@@ -40,4 +41,35 @@ describe('AsyncStorage', () => {
     await AsyncStorage.pushSelectedProfession(mockDisciplines[1])
     await expect(AsyncStorage.getSelectedProfessions()).resolves.toHaveLength(2)
   })
+
+  describe('ExerciseProgress', () => {
+    const firstExercise = { disciplineId: 1, exerciseKey: ExerciseKeys.wordChoiceExercise, score: 0.5 }
+    const secondExercise = { disciplineId: 1, exerciseKey: ExerciseKeys.writeExercise, score: 0.5 }
+    const secondExerciseBetter = { disciplineId: 1, exerciseKey: ExerciseKeys.writeExercise, score: 0.75 }
+
+    beforeEach(() => AsyncStorage.clearExerciseProgress())
```

Comment:
Is the async storage not cleaned automatically between each test yet? I think that should definitely happen. Perhaps even a jest.clearAllMocks should be enough.

Reply:
Created a ticket for this